pages/open-positions.md

```diff
@@ -5,6 +5,15 @@ redirect_from: /roles-and-teams/
 styles: assets/css/leverpostings.css
 scripts: assets/js/leverpostings.js
 ---
+
+###Important Announcement###
+On May 1, we’ll be taking down our open positions for a short application intake hiatus. During that time, we’ll be evolving how 18F plans for open roles. We estimate that we’ll post new open positions in the beginning of June. However, if that changes, we’ll update you accordingly.  
```

Comment:
How will the "updat[ing] accordingly" happen? Through this website? Through a newsletter (#57/#271)? Let's be as clear as possible about what applicants need to do to get that information.